### PR TITLE
T2.1 Image Update

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**
T2.1 Image Updates

### CODE REVIEW

File: samples/tab-graph-toolkit/nodejs/Images/10.Person1.png
Comment (Pawank-MSFT) [inline samples/tab-graph-toolkit/nodejs/Images/10.Person1.png:1]: Please hide the email address

File: samples/tab-graph-toolkit/nodejs/Images/6.PeoplePicker.png
Comment (Pawank-MSFT) [inline samples/tab-graph-toolkit/nodejs/Images/6.PeoplePicker.png:1]: hide the email addresses

File: samples/tab-graph-toolkit/nodejs/Images/8.PersonCard.png
Comment (Pawank-MSFT) [inline samples/tab-graph-toolkit/nodejs/Images/8.PersonCard.png:1]: Hide the email address

File: samples/tab-people-picker/csharp/TabPeoplePicker/Images/SetSelectedSearch.PNG
Comment (Pawank-MSFT) [inline samples/tab-people-picker/csharp/TabPeoplePicker/Images/SetSelectedSearch.PNG:1]: Hide email

File: samples/tab-staggered-permission/nodejs/Images/UserInformationCardGetPhotoAndUserMail.png
Comment (Pawank-MSFT) [inline samples/tab-staggered-permission/nodejs/Images/UserInformationCardGetPhotoAndUserMail.png:1]: Hide the emails

**Review by Pawank-MSFT — COMMENTED** [review]
some emails need to hide, please fix.

File: samples/tab-graph-toolkit/nodejs/Images/10.Person1.png
Comment (Jegadeesh-MSFT) [reply samples/tab-graph-toolkit/nodejs/Images/10.Person1.png:1]: Fixed

File: samples/tab-graph-toolkit/nodejs/Images/6.PeoplePicker.png
Comment (Jegadeesh-MSFT) [reply samples/tab-graph-toolkit/nodejs/Images/6.PeoplePicker.png:1]: Fixed

File: samples/tab-staggered-permission/nodejs/Images/UserInformationCardGetPhotoAndUserMail.png
Comment (Jegadeesh-MSFT) [reply samples/tab-staggered-permission/nodejs/Images/UserInformationCardGetPhotoAndUserMail.png:1]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8892753073

**Pull request merged**